TestFoundation/TestFileManager.swift

```diff
@@ -228,6 +228,9 @@ class TestFileManager : XCTestCase {
     }
     
     func test_fileSystemAttributes() {
+#if os(Android)
```

Comment:
Patches with tests that fail can't be merged. Would you like to instead figure out a way to mark the attributes API as not available on Android?

Reply:
[Filemanager.swift](https://github.com/SwiftJava/swift-corelibs-foundation/blob/master/Foundation/FileManager.swift#L380) marks this as NSUnimplmented so I didn’t have much option if the tests were to run. Perhaps it should be made unavailable if you can suggest the syntax for me but then what should the test do?

Reply:
If this only fails on Android, sure — it will be more noise if anyone sets up CI for this, though. In general, on platforms we CI, we'd just `#if` the test content out with an explanatory comment.

Reply:
ok, I’ll remove the XCFail as there is nothing that can be done about this.